guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -14,14 +14,30 @@ To create a file type repository in a local directory, complete the following pr
 
 . Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+** For RHEL 7:
```

Comment:
I'd always list RHEL 8 first. We should encourage users to use the latest version.